entity-framework/core/modeling/backing-field.md

```diff
@@ -7,49 +7,49 @@ uid: core/modeling/backing-field
 ---
 # Backing Fields
 
-> [!NOTE]  
-> This feature is new in EF Core 1.1.
-
 Backing fields allow EF to read and/or write to a field rather than a property. This can be useful when encapsulation in the class is being used to restrict the use of and/or enhance the semantics around access to the data by application code, but the value should be read from and/or written to the database without using those restrictions/enhancements.
 
-## Conventions
+## Basic configuration
 
-By convention, the following fields will be discovered as backing fields for a given property (listed in precedence order). Fields are only discovered for properties that are included in the model. For more information on which properties are included in the model, see [Including & Excluding Properties](included-properties.md).
+By convention, the following fields will be discovered as backing fields for a given property (listed in precedence order). 
 
 * `_<camel-cased property name>`
 * `_<property name>`
 * `m_<camel-cased property name>`
 * `m_<property name>`
 
-[!code-csharp[Main](../../../samples/core/Modeling/Conventions/BackingField.cs#Sample)]
+In the following sample, the `Url` property is configured to have `_url` as its backing field:
 
-When a backing field is configured, EF will write directly to that field when materializing entity instances from the database (rather than using the property setter). If EF needs to read or write the value at other times, it will use the property if possible. For example, if EF needs to update the value for a property, it will use the property setter if one is defined. If the property is read-only, then it will write to the field.
+[!code-csharp[Main](../../../samples/core/Modeling/Conventions/BackingField.cs#Sample)]
 
-## Data Annotations
+Note that backing fields are only discovered for properties that are included in the model. For more information on which properties are included in the model, see [Including & Excluding Properties](included-properties.md).
 
-Backing fields cannot be configured with data annotations.
+You can also configure backing fields explicitly, e.g. if the field name doesn't correspond to the above conventions:
 
-## Fluent API
+[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/BackingField.cs?name=BackingField&highlight=5)]
 
-You can use the Fluent API to configure a backing field for a property.
+## Field and property access
 
-[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/BackingField.cs#Sample)]
+By default, EF will always read and write to the backing field - assuming one has been properly configured - and will never use the property. However, EF also supports other access patterns. For example, the following sample instructs EF to write to the backing field only while materializing, and to use the property in all other cases:
```

Comment:
I think this is misleading. By default, EF will always read and write to the backing field so long as the backing field has been discovered or configured as described above. The property will be used if the backing field is not known.

Reply:
Do you think that "assuming one has been properly configured" isn't sufficient here? Maybe "assuming one has been properly discovered or configured", or something else?